pkg/network/encoding/http2.go

```diff
@@ -118,7 +121,9 @@ func (e *http2Encoder) buildAggregations(payload *network.Connections) {
 		aggregation, ok := e.aggregations[key.ConnectionKey]
 		if !ok {
 			// if there is no matching connection don't even bother to serialize HTTP2 data
-			log.Tracef("Found http2 orphan connection %v", key.ConnectionKey)
+			if log.ShouldLog(seelog.TraceLvl) {
+				log.Tracef("Found http2 orphan connection %v", key.ConnectionKey)
```

Comment:
Take a look at `log.TraceFunc` which essentially does the same thing.

Reply:
indeed, but if you're calling TraceFunc, you're allocating the parameters, that's what we're trying to avoid in hot-code-path

Reply:
ran the following:
```go
func BenchmarkTraceLog(b *testing.B) {
	b.ReportAllocs()
	b.ResetTimer()

	for i := 0; i < b.N; i++ {
		log.Trace("this is my log very long long string")
	}
}

func BenchmarkTraceLog2(b *testing.B) {
	b.ReportAllocs()
	b.ResetTimer()

	for i := 0; i < b.N; i++ {
		if log.ShouldLog(seelog.TraceLvl) {
			log.Trace("this is my log very long long string")
		}
	}
}

func BenchmarkTraceLog3(b *testing.B) {
	b.ReportAllocs()
	b.ResetTimer()

	for i := 0; i < b.N; i++ {
		log.TraceFunc(func() string {
			return "this is my log very long long string"
		})
	}
}
```

The output:
```
BenchmarkTraceLog
BenchmarkTraceLog-16     	13521062	        78.83 ns/op	      90 B/op	       2 allocs/op
BenchmarkTraceLog2
BenchmarkTraceLog2-16    	1000000000	         0.4468 ns/op	       0 B/op	       0 allocs/op
BenchmarkTraceLog3
BenchmarkTraceLog3-16    	60898656	        21.00 ns/op	      16 B/op	       1 allocs/op
```